parm/config/gfs/config.resources

```diff
@@ -440,273 +408,242 @@ case ${step} in
+        ntasks_gdas=160
+        ntasks_gfs=160
+        threads_per_task=10
         ;;
       "C192" | "C96" | "C48")
-        export npe_anal_gdas=84
-        export npe_anal_gfs=84
-        export nth_anal=5
+        ntasks_gdas=84
+        ntasks_gfs=84
+        threads_per_task=5
         ;;
       *)
         echo "FATAL ERROR: Resources not defined for job ${step} at resolution ${CASE}"
         exit 4
         ;;
     esac
-    export npe_node_anal=$(( npe_node_max / nth_anal ))
-    export nth_cycle=${nth_anal}
-    export npe_node_cycle=$(( npe_node_max / nth_cycle ))
+    tasks_per_node=$(( max_tasks_per_node / threads_per_task ))
+    export threads_per_task_cycle=${threads_per_task}
+    export tasks_per_node_cycle=$(( max_tasks_per_node / threads_per_task_cycle ))
     export is_exclusive=True
-
-    var_npe="npe_anal_${RUN}"
-    if [[ -n "${!var_npe+0}" ]]; then
-      declare -x "npe_anal"="${!var_npe}"
-    fi
     ;;
 
   "analcalc")
-    export wtime_analcalc="00:15:00"
-    export npe_analcalc=127
-    export ntasks="${npe_analcalc}"
-    export nth_analcalc=1
-    export npe_node_analcalc=$(( npe_node_max / nth_analcalc ))
-    export nth_echgres_gdas=4
-    export nth_echgres_gfs=12
+    walltime="00:15:00"
+    ntasks=127
+    export ntasks_calcanl="${ntasks}"
+    threads_per_task=1
+    tasks_per_node=$(( max_tasks_per_node / threads_per_task ))
+    export threads_per_task_echgres_gdas=4
+    export threads_per_task_echgres_gfs=12
     export is_exclusive=True
-    export memory_analcalc="48GB"
+    memory="48GB"
     if [[ "${CASE}" == "C384" || "${CASE}" == "C768" ]]; then
-       export memory_analcalc="${mem_node_max}"
-    fi
-
-    var_nth="nth_echgres_${RUN}"
-    if [[ -n "${!var_nth+0}" ]]; then
-      declare -x "nth_echgres"="${!var_nth}"
+       memory="${mem_node_max}"
     fi
     ;;
 
   "analdiag")
-    export wtime_analdiag="00:15:00"
-    export npe_analdiag=96             # Should be at least twice npe_ediag
-    export nth_analdiag=1
-    export npe_node_analdiag=$(( npe_node_max / nth_analdiag ))
-    export memory_analdiag="48GB"
+    walltime="00:15:00"
+    ntasks=96             # Should be at least twice ediag's tasks
+    threads_per_task=1
+    tasks_per_node=$(( max_tasks_per_node / threads_per_task ))
+    memory="48GB"
     ;;
 
   "sfcanl")
-    export wtime_sfcanl="00:20:00"
-    export npe_sfcanl=6
-    export nth_sfcanl=1
-    export npe_node_sfcanl=$(( npe_node_max / nth_sfcanl ))
+    walltime="00:20:00"
+    ntasks=${ntiles:-6}
+    threads_per_task=1
+    tasks_per_node=$(( max_tasks_per_node / threads_per_task ))
     export is_exclusive=True
     ;;
 
   "fcst" | "efcs")
     export is_exclusive=True
 
     if [[ "${step}" == "fcst" ]]; then
-      _CDUMP_LIST=${CDUMP:-"gdas gfs"}
+      _RUN_LIST=${RUN:-"gdas gfs"}
     elif [[ "${step}" == "efcs" ]]; then
-      _CDUMP_LIST=${CDUMP:-"enkfgdas enkfgfs"}
+      _RUN_LIST=${RUN:-"enkfgdas enkfgfs"}
     fi
```

Comment:
If the rocoto has a different config for each `RUN`, I don't think we need any of this `RUN_LIST` stuff anymore.

Reply:
I think you're right.  I'll test this out.

Reply:
Fixed.